### PR TITLE
Add imu support

### PR DESCRIPTION
**Pull request by MaximilienNaveau**
[//]: # "Thanks for your contribution.  To make life of the reviewers easier,"
[//]: # "please give this pull request a meaningful title and provide the"
[//]: # "requested information below."

## Description

[//]: # "Description of what you did.  If it is more complex, consider to add a"
[//]: # "'Summary' section."

Add the imu support.

## How I Tested

[//]: # "Explain how you tested your changes"

To be tested on the robot.

## I fulfilled the following requirements

[//]: # "Please make sure you followed these steps before requesting a review."
[//]: # "Check the boxes in the list below, when done."

- [ ] All new code is formatted according to our style guide (for C++ run clang-format, for Python, run flake8 and fix all warnings).
- [ ] All new functions/classes are documented and existing documentation is updated according to changes.
- [ ] No commented code from testing/debugging is kept (unless there is a good reason to keep it).

### CODE REVIEW

**Comment by MaximilienNaveau** [comment]
I will do clang-format after this PR is accepted.

File: include/blmc_robots/solo12.hpp [hunk 1]
```diff
@@ -206,7 +209,62 @@ class Solo12
...
+        return imu_linear_acceleration_;
+    }
+
+    /** @brief base attitude quaternion from imu.
+     * @return
+     * WARNING !!!!
+     * The method <acquire_sensors>"()" has to be called
+     * prior to any getter to have up to date data.
+     */
+    const Eigen::Ref<Eigen::Vector4d> get_imu_attitude_quaternion()
```
Comment (jviereck) [inline include/blmc_robots/solo12.hpp:262]: Would it make sense to document the ordering of the quaternion in the 4 vector? E.g. is it `xyzw` or `wxyz` order.

**Review by jviereck — APPROVED** [review]

File: include/blmc_robots/solo12.hpp (hunk 1 above)
Comment (MaximilienNaveau) [reply include/blmc_robots/solo12.hpp:262]: Done :+1:

**Comment by jviereck** [comment]
This PR is ready for landing. @MaximilienNaveau I let you do the merging as I am not sure you want to test this on the real hardware first or not.

**Comment by MaximilienNaveau** [comment]
Thanks @jviereck I will indeed test this on the robot before merging :+1:

**Pull request merged**